app/templates/ask.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="row h-screen ">
+  <main role="main" class="col-12">
```

Comment:
Why is there a space in `class` value? What happened to 4-space indentation? Also Bootstrap's `.row > .col-X-12` does nothing. You can easily avoid 1 wrapping element.